discord_bots/utils.py

```diff
@@ -825,3 +829,174 @@ def get_team_name_diff(
+                    colour=Colour.red(),
+                )
+                return
+            return
+
+        team0_ipg_players: list[InProgressGamePlayer] = session.query(
+            InProgressGamePlayer
+        ).filter(
+            InProgressGamePlayer.in_progress_game_id == in_progress_game.id,
+            InProgressGamePlayer.team == 0,
+        )
+        team1_ipg_players: list[InProgressGamePlayer] = session.query(
+            InProgressGamePlayer
+        ).filter(
+            InProgressGamePlayer.in_progress_game_id == in_progress_game.id,
+            InProgressGamePlayer.team == 1,
+        )
+        team0_player_ids = set(map(lambda x: x.player_id, team0_ipg_players))
+        team1_player_ids = set(map(lambda x: x.player_id, team1_ipg_players))
+        team0_players: list[Player] = session.query(Player).filter(Player.id.in_(team0_player_ids))  # type: ignore
+        team1_players: list[Player] = session.query(Player).filter(Player.id.in_(team1_player_ids))  # type: ignore
+
+        be_voice_channel: VoiceChannel | None = None
+        ds_voice_channel: VoiceChannel | None = None
+        ipg_channels: list[InProgressGameChannel] | None = (
+            session.query(InProgressGameChannel)
+            .filter(InProgressGameChannel.in_progress_game_id == in_progress_game.id)
+            .all()
+        )
+        for ipg_channel in ipg_channels or []:
+            discord_channel: discord.abc.GuildChannel | None = guild.get_channel(
+                ipg_channel.channel_id
+            )
+            if isinstance(discord_channel, VoiceChannel):
+                # This is suboptimal solution but it's good enough for now. We should keep track of each team's VC in the database
+                if discord_channel.name == in_progress_game.team0_name:
+                    be_voice_channel = discord_channel
+                elif discord_channel.name == in_progress_game.team1_name:
+                    ds_voice_channel = discord_channel
+
+        coroutines = []
+        for player in team0_players:
+            if player.move_enabled and be_voice_channel:
+                member: Member | None = guild.get_member(player.id)
+                if member:
+                    member_voice: VoiceState | None = member.voice
+                    if member_voice:
+                        if member_voice.channel:
+                            try:
+                                coroutines.append(
+                                    member.move_to(
+                                        be_voice_channel,
+                                        reason=f"Game {game_id} started",
+                                    )
+                                )
+                            except Exception:
+                                _log.exception(
+                                    f"Caught exception moving player to voice channel"
+                                )
+
+        for player in team1_players:
```

Comment:
you don't need to do this, but it would be nice to just merge these two loops into a single nested function call. Not entirely necessary though, but it would reduce a decent amount of lines. Can always just throw a TODO in there